src/main/resources/swagger.api/schema/resource/response/WorkResponse.json

```diff
@@ -12,7 +12,7 @@
           "type": "array",
           "items": {
             "type": "object",
-            "$ref": "title/TitleField.json"
+            "$ref": "../common/title/TitleField.json"
           },
           "x-json-property": "http://bibfra.me/vocab/marc/title",
           "x-field-extra-annotation": "@org.folio.linked.data.validation.PrimaryTitleConstraint"
```

Comment:
Should we add the constraint in the response? If we decide to add it here, can we make the same change in InstanceResponse.json also?

Reply:
nope, it's a mistake - there is no point to mark response DTO with validation constraints, since there is no functionality for this and it just doesn't make sense. Removed, thank you!